examples/12_rerank.yaml

```diff
@@ -0,0 +1,75 @@
+# Using a trained model to do reranking score for an nbest list
+
```

Comment:
Maybe make this a little bit more verbose? What is the location of the existing model, where is the n-best list? What will be the expected output after running this file?

Also, perhaps we should make a distinction in the terminology. Are we doing reranking (taking in an n-best list and picking the best one) or hypothesis scoring (taking in the hypotheses and assigning a score to each)? I think both are useful, so perhaps we can distinguish between the two.